bin/openosp-setup.sh

```diff
@@ -128,7 +137,17 @@ ARCH=`uname -m`
 if [[ $ARCH == 'arm64' || $ARCH == 'aarch64' ]]; then
   echo "Not pulling pre-built docker images as you are on an ARM machine; you'll need to build them yourself."
 else
-  docker pull openosp/open-osp
-  docker pull openosp/expedius
-  docker pull openosp/faxws
+  if [[ $* == *--noinput* ]]; then
```

Comment:
You may want to seriously consider keeping an OSCAR 19 build on Docker hub.  It will make building, versioning , and updating infinitely easier.